Retrieving logs for app [36;1m1chuu_sharehouseProj2018[0m in org [36;1mdnjscjf92@likelion.org[0m / space [36;1mdev[0m as [36;1mdnjscjf92@likelion.org[0m...

   2018-03-30T08:03:34.39+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T08:04:01.18+0900 [CELL/0] OUT Stopping instance 379e7254-1eca-43b5-6617-971b
   2018-03-30T08:04:01.18+0900 [CELL/0] OUT Destroying container
   2018-03-30T08:20:32.10+0900 [CELL/0] OUT Successfully created container
   2018-03-30T08:20:42.05+0900 [APP/PROC/WEB/0] OUT > node server.js
   [31m2018-03-30T08:20:43.94+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T08:20:43.94+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T08:20:43.94+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T08:20:43.96+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   2018-03-30T08:20:44.09+0900 [API/5] OUT Process has crashed with type: "web"
   2018-03-30T08:20:44.10+0900 [API/5] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"144a4a7f-0b5e-4434-70f3-4873", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>43, "crash_timestamp"=>1522365644044023412, "version"=>"db010fa4-d9e9-4afb-ba45-e525ed7bcaab"}
   2018-03-30T08:37:21.87+0900 [CELL/0] OUT Creating container
   2018-03-30T08:37:23.20+0900 [CELL/0] OUT Successfully created container
   2018-03-30T08:37:29.67+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T08:37:29.67+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T08:37:30.86+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T08:37:30.92+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T08:37:30.92+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T08:37:30.92+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T08:37:30.92+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T08:37:30.92+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T08:37:30.95+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   2018-03-30T08:37:31.00+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T08:37:31.08+0900 [CELL/0] OUT Destroying container
   2018-03-30T08:37:31.14+0900 [API/5] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"6fb10943-4d90-4e53-5a08-bfc1", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>44, "crash_timestamp"=>1522366651063115370, "version"=>"db010fa4-d9e9-4afb-ba45-e525ed7bcaab"}
   2018-03-30T08:37:32.53+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T08:54:06.30+0900 [CELL/0] OUT Creating container
   2018-03-30T08:54:12.97+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T08:54:13.96+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T08:54:14.34+0900 [APP/PROC/WEB/0] OUT APIs initialize
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at errnoException (dns.js:28:10)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T08:54:15.24+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T08:54:15.25+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T08:54:15.25+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T08:54:15.25+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T08:54:15.25+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T08:54:15.27+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T08:54:15.27+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-29T23_54_15_260Z-debug.log[0m
   2018-03-30T08:54:15.29+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T08:54:15.31+0900 [CELL/0] OUT Exit status 0
   2018-03-30T08:54:15.32+0900 [CELL/0] OUT Stopping instance 2e9937b0-4d70-495b-6e37-7588
   2018-03-30T08:54:15.32+0900 [CELL/0] OUT Destroying container
   2018-03-30T08:54:15.40+0900 [API/3] OUT Process has crashed with type: "web"
   2018-03-30T08:54:15.41+0900 [API/3] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"2e9937b0-4d70-495b-6e37-7588", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>45, "crash_timestamp"=>1522367655324593334, "version"=>"db010fa4-d9e9-4afb-ba45-e525ed7bcaab"}
   2018-03-30T09:10:33.27+0900 [CELL/0] OUT Creating container
   2018-03-30T09:10:34.71+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:10:40.11+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:10:40.89+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:10:40.89+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:10:41.22+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR Error: getaddrinfo ENOTFOUND cloud_instance_private_ip cloud_instance_private_ip:3306[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at errnoException (dns.js:28:10)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:10:41.99+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:10:42.01+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:10:42.02+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:10:42.02+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_10_42_016Z-debug.log[0m
   2018-03-30T09:10:42.05+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:10:42.06+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:10:42.07+0900 [CELL/0] OUT Stopping instance 65efbf02-1c5a-45fc-7c0a-521e
   2018-03-30T09:10:42.07+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:10:42.15+0900 [API/3] OUT Process has crashed with type: "web"
   2018-03-30T09:10:42.24+0900 [API/3] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"65efbf02-1c5a-45fc-7c0a-521e", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>46, "crash_timestamp"=>1522368642080677971, "version"=>"db010fa4-d9e9-4afb-ba45-e525ed7bcaab"}
   2018-03-30T09:10:43.49+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:18:27.63+0900 [API/2] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"name"=>"1chuu_sharehouseProj2018", "command"=>"PRIVATE DATA HIDDEN", "instances"=>1, "memory"=>256, "environment_json"=>"PRIVATE DATA HIDDEN"})
   2018-03-30T09:19:06.47+0900 [API/5] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"state"=>"STOPPED"})
   2018-03-30T09:19:10.35+0900 [API/3] OUT Creating build for app with guid 384cc462-484d-48c3-882d-577584bb9914
   2018-03-30T09:19:11.92+0900 [API/3] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"state"=>"STARTED"})
   2018-03-30T09:19:12.74+0900 [STG/0] OUT Downloading liberty_v3_14-20171013-1023...
   2018-03-30T09:19:12.75+0900 [STG/0] OUT Downloading liberty-for-java...
   2018-03-30T09:19:12.75+0900 [STG/0] OUT Downloading sdk-for-nodejs...
   2018-03-30T09:19:12.75+0900 [STG/0] OUT Downloading dotnet-core...
   2018-03-30T09:19:12.75+0900 [STG/0] OUT Downloading swift_buildpack...
   2018-03-30T09:19:12.76+0900 [STG/0] OUT Downloaded liberty_v3_14-20171013-1023
   2018-03-30T09:19:12.76+0900 [STG/0] OUT Downloading noop-buildpack...
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloaded dotnet-core
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloading java_buildpack...
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloaded liberty-for-java
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloading ruby_buildpack...
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloaded sdk-for-nodejs
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloading nodejs_buildpack...
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloaded swift_buildpack
   2018-03-30T09:19:12.77+0900 [STG/0] OUT Downloading go_buildpack...
   2018-03-30T09:19:12.78+0900 [STG/0] OUT Downloaded noop-buildpack
   2018-03-30T09:19:12.78+0900 [STG/0] OUT Downloading python_buildpack...
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloaded java_buildpack
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloading xpages_buildpack...
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloaded ruby_buildpack
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloading php_buildpack...
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloaded nodejs_buildpack
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloading staticfile_buildpack...
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloaded python_buildpack
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloading binary_buildpack...
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloaded go_buildpack
   2018-03-30T09:19:12.79+0900 [STG/0] OUT Downloading swift_buildpack_v2_0_9-20171112-1715...
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloaded php_buildpack
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloading liberty-for-java_v3_17_1-20180131-1532...
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloaded staticfile_buildpack
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloading liberty-for-java_v3_18-20180213-1234...
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloaded xpages_buildpack
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloading sdk-for-nodejs_v3_18-20180206-1137...
   2018-03-30T09:19:12.81+0900 [STG/0] OUT Downloaded binary_buildpack
   2018-03-30T09:19:12.82+0900 [STG/0] OUT Downloaded swift_buildpack_v2_0_9-20171112-1715
   2018-03-30T09:19:12.82+0900 [STG/0] OUT Downloaded liberty-for-java_v3_17_1-20180131-1532
   2018-03-30T09:19:12.82+0900 [STG/0] OUT Downloaded liberty-for-java_v3_18-20180213-1234
   2018-03-30T09:19:12.83+0900 [STG/0] OUT Downloaded sdk-for-nodejs_v3_18-20180206-1137
   2018-03-30T09:19:12.83+0900 [STG/0] OUT Creating container
   2018-03-30T09:19:13.98+0900 [STG/0] OUT Successfully created container
   2018-03-30T09:19:13.98+0900 [STG/0] OUT Downloading app package...
   2018-03-30T09:19:14.12+0900 [STG/0] OUT Downloaded app package (1.6M)
   2018-03-30T09:19:14.12+0900 [STG/0] OUT Downloading build artifacts cache...
   2018-03-30T09:19:16.49+0900 [STG/0] OUT Downloaded build artifacts cache (10.3M)
   2018-03-30T09:19:16.49+0900 [STG/0] OUT Staging...
   2018-03-30T09:19:19.09+0900 [STG/0] OUT -----> IBM SDK for Node.js Buildpack v3.19-20180314-1533
   2018-03-30T09:19:19.10+0900 [STG/0] OUT        Based on Cloud Foundry Node.js Buildpack v1.5.24
   2018-03-30T09:19:19.11+0900 [STG/0] OUT -----> Creating runtime environment
   2018-03-30T09:19:19.12+0900 [STG/0] OUT        
   2018-03-30T09:19:19.12+0900 [STG/0] OUT        NPM_CONFIG_LOGLEVEL=error
   2018-03-30T09:19:19.12+0900 [STG/0] OUT        NPM_CONFIG_PRODUCTION=true
   2018-03-30T09:19:19.13+0900 [STG/0] OUT        NODE_ENV=production
   2018-03-30T09:19:19.13+0900 [STG/0] OUT        NODE_MODULES_CACHE=true
   2018-03-30T09:19:19.13+0900 [STG/0] OUT -----> Installing binaries
   2018-03-30T09:19:19.17+0900 [STG/0] OUT        engines.node (package.json):  >= 6.9.x
   2018-03-30T09:19:19.17+0900 [STG/0] OUT        engines.npm (package.json):   > 3.10.x
   2018-03-30T09:19:19.17+0900 [STG/0] OUT        
   2018-03-30T09:19:19.17+0900 [STG/0] OUT        Resolving node version >= 6.9.x via 'node-version-resolver'
   2018-03-30T09:19:19.52+0900 [STG/0] OUT        Installing IBM SDK for Node.js (6.13.0) from cache
   2018-03-30T09:19:22.18+0900 [STG/0] OUT        Resolving npm version > 3.10.x via semver.io...
   2018-03-30T09:19:23.49+0900 [STG/0] OUT        Downloading and installing npm 5.8.0 (replacing version 3.10.10)...
   2018-03-30T09:20:05.06+0900 [STG/0] OUT -----> Restoring cache
   2018-03-30T09:20:05.73+0900 [STG/0] OUT        Loading 2 from cacheDirectories (default):
   2018-03-30T09:20:05.73+0900 [STG/0] OUT        - node_modules
   2018-03-30T09:20:05.73+0900 [STG/0] OUT        - bower_components (not cached - skipping)
   2018-03-30T09:20:05.73+0900 [STG/0] OUT -----> Checking and configuring service extensions before installing dependencies
   2018-03-30T09:20:06.14+0900 [STG/0] OUT -----> Building dependencies
   2018-03-30T09:20:06.14+0900 [STG/0] OUT        Installing node modules (package.json)
   2018-03-30T09:20:10.88+0900 [STG/0] OUT        up to date in 3.583s
   2018-03-30T09:20:10.92+0900 [STG/0] OUT -----> Checking and configuring service extensions after installing dependencies
   2018-03-30T09:20:11.41+0900 [STG/0] OUT -----> Installing App Management
   2018-03-30T09:20:24.50+0900 [STG/0] OUT -----> Caching build
   2018-03-30T09:20:24.52+0900 [STG/0] OUT        Clearing previous node cache
   2018-03-30T09:20:24.53+0900 [STG/0] OUT        Saving 2 cacheDirectories (default):
   2018-03-30T09:20:24.53+0900 [STG/0] OUT        - node_modules
   2018-03-30T09:20:29.54+0900 [STG/0] OUT        - bower_components (nothing to cache)
   2018-03-30T09:20:30.20+0900 [STG/0] OUT -----> Build succeeded!
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── body-parser@1.18.2
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── cfenv@1.0.4
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── cloudant@1.10.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── dotenv@2.0.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── express@4.16.3
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── google-auth-library@0.11.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── googleapis@22.2.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── json2csv@3.11.5
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── moment@2.21.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── moment-timezone@0.5.14
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── mysql@2.15.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── request@2.85.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        ├── tinyurl@1.1.2
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        └── watson-developer-cloud@2.42.0
   2018-03-30T09:20:33.11+0900 [STG/0] OUT        
   2018-03-30T09:20:56.16+0900 [STG/0] OUT Exit status 0
   2018-03-30T09:20:56.16+0900 [STG/0] OUT Staging complete
   2018-03-30T09:20:56.16+0900 [STG/0] OUT Uploading droplet, build artifacts cache...
   2018-03-30T09:20:56.16+0900 [STG/0] OUT Uploading build artifacts cache...
   2018-03-30T09:20:56.16+0900 [STG/0] OUT Uploading droplet...
   2018-03-30T09:20:56.55+0900 [STG/0] OUT Uploaded build artifacts cache (10.3M)
   2018-03-30T09:20:57.26+0900 [API/2] OUT Creating droplet for app with guid 384cc462-484d-48c3-882d-577584bb9914
   2018-03-30T09:21:01.45+0900 [STG/0] OUT Uploaded droplet (34.5M)
   2018-03-30T09:21:01.47+0900 [STG/0] OUT Uploading complete
   2018-03-30T09:21:01.55+0900 [STG/0] OUT Stopping instance 56ece9d4-53ca-48e5-81f0-cbdb0a118032
   2018-03-30T09:21:01.55+0900 [STG/0] OUT Destroying container
   2018-03-30T09:21:02.84+0900 [CELL/0] OUT Creating container
   2018-03-30T09:21:03.79+0900 [STG/0] OUT Successfully destroyed container
   2018-03-30T09:21:04.62+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:21:10.43+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:21:11.36+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:21:11.36+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:21:11.69+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:21:12.64+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   2018-03-30T09:21:12.69+0900 [CELL/0] OUT Container became healthy
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:21:12.73+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:21:12.76+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:21:12.78+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:21:12.78+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_21_12_772Z-debug.log[0m
   2018-03-30T09:21:12.80+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:21:12.81+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:21:12.89+0900 [CELL/0] OUT Stopping instance a57bf20c-fee8-42bf-607a-fb15
   2018-03-30T09:21:12.89+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:21:12.91+0900 [API/1] OUT Process has crashed with type: "web"
   2018-03-30T09:21:12.93+0900 [API/1] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"a57bf20c-fee8-42bf-607a-fb15", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>1, "crash_timestamp"=>1522369272843124813, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:21:13.58+0900 [CELL/0] OUT Creating container
   2018-03-30T09:21:14.62+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:21:14.65+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:21:25.40+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:21:28.27+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:21:28.27+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:21:29.00+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:21:32.49+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:21:33.22+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:21:33.79+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:21:33.79+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:21:33.90+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:21:33.90+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:21:33.90+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:21:33.91+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:21:33.91+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:21:34.36+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:21:34.36+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_21_34_004Z-debug.log[0m
   2018-03-30T09:21:34.56+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:21:34.57+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:21:34.62+0900 [CELL/0] OUT Stopping instance eea4f654-5c78-414e-4628-a982
   2018-03-30T09:21:34.62+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:21:34.64+0900 [API/4] OUT Process has crashed with type: "web"
   2018-03-30T09:21:34.66+0900 [API/4] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"eea4f654-5c78-414e-4628-a982", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>2, "crash_timestamp"=>1522369294584370289, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:21:35.31+0900 [CELL/0] OUT Creating container
   2018-03-30T09:21:37.00+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:21:37.94+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:21:45.02+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:22:10.19+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:22:10.19+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:22:22.08+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:22:30.18+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   2018-03-30T09:22:30.48+0900 [CELL/0] OUT Container became healthy
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:22:31.25+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:22:31.60+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:22:31.68+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:22:31.77+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:22:31.77+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_22_31_705Z-debug.log[0m
   2018-03-30T09:22:31.84+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:22:31.89+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:22:31.90+0900 [CELL/0] OUT Stopping instance 27f6ac2e-6623-4f0d-6cd7-6711
   2018-03-30T09:22:31.90+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:22:31.92+0900 [API/1] OUT Process has crashed with type: "web"
   2018-03-30T09:22:31.94+0900 [API/1] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"27f6ac2e-6623-4f0d-6cd7-6711", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>3, "crash_timestamp"=>1522369351902762851, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:22:33.36+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:23:14.93+0900 [CELL/0] OUT Creating container
   2018-03-30T09:23:16.67+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:23:24.82+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:23:26.24+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:23:26.24+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:23:26.62+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:23:27.60+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:23:27.67+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:23:27.78+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:23:27.80+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:23:27.80+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_23_27_787Z-debug.log[0m
   2018-03-30T09:23:27.83+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:23:27.85+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:23:27.86+0900 [CELL/0] OUT Stopping instance ec0cc695-969b-4b23-6230-dc3e
   2018-03-30T09:23:27.86+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:23:28.31+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:23:28.60+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"ec0cc695-969b-4b23-6230-dc3e", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>4, "crash_timestamp"=>1522369407862661281, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:23:29.79+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:24:58.42+0900 [CELL/0] OUT Creating container
   2018-03-30T09:25:00.33+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:25:07.13+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:25:08.58+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:25:08.58+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:25:09.09+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:25:10.04+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:25:10.12+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:25:10.18+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_25_10_179Z-debug.log[0m
   2018-03-30T09:25:10.21+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:25:10.22+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:25:10.24+0900 [CELL/0] OUT Stopping instance 2d56c9aa-5b6f-4b64-6867-70f5
   2018-03-30T09:25:10.24+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:25:10.54+0900 [API/1] OUT Process has crashed with type: "web"
   2018-03-30T09:25:10.68+0900 [API/1] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"2d56c9aa-5b6f-4b64-6867-70f5", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>5, "crash_timestamp"=>1522369510234322744, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:25:12.05+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:27:17.32+0900 [CELL/0] OUT Creating container
   2018-03-30T09:27:18.78+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:27:25.70+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:27:26.60+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:27:26.60+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:27:26.93+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:27:27.89+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:27:27.97+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:27:28.02+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:27:28.02+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:27:28.04+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:27:28.04+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:27:28.04+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:27:28.04+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:27:28.04+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:27:28.10+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:27:28.10+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_27_28_057Z-debug.log[0m
   2018-03-30T09:27:28.15+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:27:28.18+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:27:28.19+0900 [CELL/0] OUT Stopping instance 0a25020d-52d7-4cef-4882-0a24
   2018-03-30T09:27:28.19+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:27:28.23+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:27:28.25+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"0a25020d-52d7-4cef-4882-0a24", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>6, "crash_timestamp"=>1522369648198942524, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:27:29.91+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:31:55.22+0900 [CELL/0] OUT Creating container
   2018-03-30T09:31:56.35+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:32:00.89+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:32:01.69+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:32:01.69+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:32:01.96+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:32:02.63+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:32:02.67+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:32:02.71+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:32:02.71+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:32:02.72+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_32_02_721Z-debug.log[0m
   2018-03-30T09:32:02.76+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:32:02.77+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:32:03.05+0900 [CELL/0] OUT Stopping instance 0ca03e15-8597-4be6-49e3-2c1b
   2018-03-30T09:32:03.05+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:32:03.84+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:32:03.93+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"0ca03e15-8597-4be6-49e3-2c1b", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>7, "crash_timestamp"=>1522369922949156816, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:32:04.21+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:40:41.27+0900 [CELL/0] OUT Creating container
   2018-03-30T09:40:42.29+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:40:47.60+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:40:48.63+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:40:48.63+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:40:48.91+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:40:49.61+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:40:49.66+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:40:49.70+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:40:49.71+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:40:49.71+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_40_49_710Z-debug.log[0m
   2018-03-30T09:40:49.75+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:40:49.76+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:40:49.78+0900 [CELL/0] OUT Stopping instance 3e0ef04b-45d7-41dc-47a5-7fc7
   2018-03-30T09:40:49.78+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:40:50.01+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:40:50.20+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"3e0ef04b-45d7-41dc-47a5-7fc7", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>8, "crash_timestamp"=>1522370449774427111, "version"=>"6f9c69a5-335e-4243-a033-9c7b43d87c7a"}
   2018-03-30T09:40:51.30+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:45:53.34+0900 [API/4] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"name"=>"1chuu_sharehouseProj2018", "command"=>"PRIVATE DATA HIDDEN", "instances"=>1, "memory"=>256, "environment_json"=>"PRIVATE DATA HIDDEN"})
   2018-03-30T09:46:15.96+0900 [API/1] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"state"=>"STOPPED"})
   2018-03-30T09:46:19.88+0900 [API/1] OUT Creating build for app with guid 384cc462-484d-48c3-882d-577584bb9914
   2018-03-30T09:46:21.32+0900 [API/1] OUT Updated app with guid 384cc462-484d-48c3-882d-577584bb9914 ({"state"=>"STARTED"})
   2018-03-30T09:47:57.05+0900 [API/1] OUT Creating droplet for app with guid 384cc462-484d-48c3-882d-577584bb9914
   2018-03-30T09:48:00.22+0900 [CELL/0] OUT Creating container
   2018-03-30T09:48:01.37+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:48:06.95+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:48:08.10+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:48:08.10+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:48:08.41+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:48:09.22+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:48:09.27+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:48:09.29+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:48:09.29+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:48:09.30+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:48:09.30+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:48:09.30+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:48:09.30+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:48:09.30+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:48:09.31+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:48:09.31+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_48_09_306Z-debug.log[0m
   2018-03-30T09:48:09.33+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:48:09.33+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:48:09.87+0900 [CELL/0] OUT Stopping instance 122763d8-0411-493c-4f24-66f2
   2018-03-30T09:48:09.87+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:48:09.89+0900 [API/5] OUT Process has crashed with type: "web"
   2018-03-30T09:48:09.99+0900 [API/5] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"122763d8-0411-493c-4f24-66f2", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>1, "crash_timestamp"=>1522370889643599264, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:48:10.76+0900 [CELL/0] OUT Creating container
   2018-03-30T09:48:11.04+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:48:12.15+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:48:20.22+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:48:21.05+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:48:21.05+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:48:21.33+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:48:22.01+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:48:22.04+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:48:22.07+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_48_22_073Z-debug.log[0m
   2018-03-30T09:48:22.09+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:48:22.09+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:48:22.12+0900 [CELL/0] OUT Stopping instance 5c8a0f45-d1f3-470b-4e8d-581c
   2018-03-30T09:48:22.12+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:48:22.90+0900 [CELL/0] OUT Creating container
   2018-03-30T09:48:23.31+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:48:24.43+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:48:27.83+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:48:27.85+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"5c8a0f45-d1f3-470b-4e8d-581c", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>2, "crash_timestamp"=>1522370902097993732, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:48:38.86+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:48:39.62+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:48:39.62+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:48:39.93+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:48:40.68+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:48:40.75+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:48:40.81+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:48:40.82+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:48:40.93+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:48:40.93+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_48_40_823Z-debug.log[0m
   2018-03-30T09:48:40.95+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:48:40.97+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:48:40.98+0900 [CELL/0] OUT Stopping instance 9632137b-5490-4e39-56cd-ad4a
   2018-03-30T09:48:40.98+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:48:41.02+0900 [API/2] OUT Process has crashed with type: "web"
   2018-03-30T09:48:41.07+0900 [API/2] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"9632137b-5490-4e39-56cd-ad4a", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>3, "crash_timestamp"=>1522370920983465171, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:48:42.29+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:49:17.23+0900 [CELL/0] OUT Creating container
   2018-03-30T09:49:20.84+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:49:33.32+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:49:38.36+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:49:38.36+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:49:42.10+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:49:46.89+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:49:46.99+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:49:47.04+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:49:47.06+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:49:47.16+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:49:47.16+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_49_47_067Z-debug.log[0m
   2018-03-30T09:49:47.19+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:49:47.20+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:49:47.22+0900 [CELL/0] OUT Stopping instance ca62cf33-5e7f-4c0c-59f7-8ae6
   2018-03-30T09:49:47.22+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:49:47.24+0900 [API/0] OUT Process has crashed with type: "web"
   2018-03-30T09:49:47.26+0900 [API/0] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"ca62cf33-5e7f-4c0c-59f7-8ae6", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>4, "crash_timestamp"=>1522370987218161889, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:49:49.82+0900 [CELL/0] OUT Successfully destroyed container
   2018-03-30T09:51:18.56+0900 [API/3] OUT Process has crashed with type: "web"
   2018-03-30T09:51:18.57+0900 [API/3] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"5cad0c39-95d3-49db-7de4-e9bf", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>5, "crash_timestamp"=>1522371078534336575, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:53:27.32+0900 [CELL/0] OUT Creating container
   2018-03-30T09:53:29.26+0900 [CELL/0] OUT Successfully created container
   2018-03-30T09:53:38.94+0900 [CELL/0] OUT Starting health monitoring of container
   2018-03-30T09:53:41.19+0900 [APP/PROC/WEB/0] OUT > conversation-simple@0.1.1 start /home/vcap/app
   2018-03-30T09:53:41.19+0900 [APP/PROC/WEB/0] OUT > node server.js
   2018-03-30T09:53:41.90+0900 [APP/PROC/WEB/0] OUT APIs initialize
   2018-03-30T09:53:44.80+0900 [APP/PROC/WEB/0] OUT Server running on port: 8080
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR /home/vcap/app/api/message.js:42[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR   if (err) throw err;[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR            ^[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR Error: connect ECONNREFUSED 127.0.0.1:3306[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Object.exports._errnoException (util.js:1020:11)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at exports._exceptionWithHostPort (util.js:1043:20)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     --------------------[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Protocol._enqueue (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Protocol.handshake (/home/vcap/app/node_modules/mysql/lib/protocol/Protocol.js:52:23)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Connection.connect (/home/vcap/app/node_modules/mysql/lib/Connection.js:130:18)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/api/message.js:32:13)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:577:32)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:586:10)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Module.load (module.js:494:32)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:453:12)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:445:3)[0m
   [31m2018-03-30T09:53:44.94+0900 [APP/PROC/WEB/0] ERR     at Module.require (module.js:504:17)[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! errno 1[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! conversation-simple@0.1.1 start: `node server.js`[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! [0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! Failed at the conversation-simple@0.1.1 start script.[0m
   [31m2018-03-30T09:53:45.04+0900 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.[0m
   [31m2018-03-30T09:53:45.23+0900 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:[0m
   [31m2018-03-30T09:53:45.23+0900 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2018-03-30T00_53_45_130Z-debug.log[0m
   2018-03-30T09:53:45.33+0900 [APP/PROC/WEB/0] OUT Exit status 1
   2018-03-30T09:53:45.36+0900 [CELL/0] OUT Exit status 0
   2018-03-30T09:53:45.41+0900 [CELL/0] OUT Stopping instance 3b05bc57-b829-46c7-5899-f46b
   2018-03-30T09:53:45.41+0900 [CELL/0] OUT Destroying container
   2018-03-30T09:53:45.43+0900 [API/1] OUT Process has crashed with type: "web"
   2018-03-30T09:53:45.48+0900 [API/1] OUT App instance exited with guid 384cc462-484d-48c3-882d-577584bb9914 payload: {"instance"=>"3b05bc57-b829-46c7-5899-f46b", "index"=>0, "reason"=>"CRASHED", "exit_description"=>"APP/PROC/WEB: Exited with status 1", "crash_count"=>6, "crash_timestamp"=>1522371225360881395, "version"=>"efb7ef3d-580b-448d-abf6-3b2af0a14158"}
   2018-03-30T09:53:48.27+0900 [CELL/0] OUT Successfully destroyed container
